src/main/java/com/misterveiga/cds/utils/EmbedBuilds.java

```diff
@@ -168,7 +168,7 @@ public static EmbedBuilder scanUrl(String url, String botIcon) throws Interrupte
 		embed.setAuthor("Malicious Log", null, botIcon);
 
 		if (positives == 0) {
-			embed.setTitle("Safe Link Detected");
+			embed.setTitle("No Malicious Content Detected");
 			embed.setThumbnail("https://icons.iconarchive.com/icons/paomedia/small-n-flat/512/sign-check-icon.png");
```

Comment:
Maybe we should replace this with a yellow question mark instead of a green check.